index.html

```diff
@@ -40,20 +40,21 @@
   });
 
   // Add a callback to the serach component
-  document.body.addEventListener('myuw-search', function(event) {
-    var value = event.detail.value;
-    var label = document.getElementById('searchLabel');
-    var searchValue = document.getElementById('searchValue');
-    label.innerText = 'You searched for: ';
-    searchValue.innerText = '"' + value + '"';
+  document.body.addEventListener('myuw-search', function (event) {
```

Comment:
So, for serious, what is this style that's being automatically applied, how would I replicate this behavior, why is this correct style, and what can we do to document this?

I'm going to continue to make an effort to not have my own opinions about JavaScript code style. If this is correct style, I can roll with it. But I'd like to at least write down what it is.

In [other contexts](https://github.com/uPortal-Project/uportal-home/blob/master/.github/CONTRIBUTING.md#google-code-style), MyUW has [tried](https://apetro.ghost.io/citizens-for-a-better-uportal-tomorrow-tomorrow/#adoptgooglestyle) to adopt Google Style. I read Google Style for JavaScript as [prescribing](https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace) including the whitespaces around the equals (  ` = `, it being a "binary operator"), and prescribing `function(event)` rather than an internal space between function and the paren.

Maybe this project shouldn't adopt Google Style. If so, what style guide is it adopting, let's write that down, and preferably automate it, in a way that I too can readily share in the automation.